faq.html

```diff
@@ -203,12 +202,12 @@ <h4 class="panel-title">
               <div class="panel panel-default">
                 <div class="panel-heading">
                   <h4 class="panel-title">
-                    <a class="accordion-toggle collapsed" data-toggle="collapse" data-parent="#accordion" href="#P-3">Why is AntennaPod not on the &lt;Foo&gt; Store?</a>
+                    <a class="accordion-toggle collapsed" data-toggle="collapse" data-parent="#accordion" href="#P-3">Why is AntennaPod not on Store X or Y?</a>
                   </h4>
                 </div>
                 <div id="P-3" class="panel-collapse collapse">
                   <div class="panel-body">
-                    Frankly, we don't have the time to support other stores besides Google Play. If nothing else, you can get AntennaPod on <a href="https://f-droid.org/repository/browse/?fdfilter=antennapod&fdid=de.danoeh.antennapod">F-Droid</a>.
+                    Unfortunately, we don't have the time to support other stores besides Google Play. If nothing else, you can get AntennaPod on <a href="https://f-droid.org/repository/browse/?fdfilter=antennapod&fdid=de.danoeh.antennapod">F-Droid</a>.
```

Comment:
Sorry, I'm a perfectionist, so I often nitpick indeed (while equally often I don't see it as such).
Didn't know `&` should be replaced in a url, but now I'm at it, I will remove the `fdfilter=antennapod` which also makes the `&` unnecessary ;)